<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>581</height>
     </rect>
    </property>
    <widget class="QPushButton" name="enterbut">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>6</y>
       <width>141</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="varbut">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>51</y>
       <width>141</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>Variables</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearbut">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>96</y>
       <width>141</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inptxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>771</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="restxt">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>380</y>
       <width>391</width>
       <height>161</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>771</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="plotbut">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>141</y>
       <width>141</width>
       <height>55</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
    </widget>
    <widget class="QWidget" name="plotWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-150</x>
       <y>10</y>
       <width>881</width>
       <height>611</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="inpLine">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>581</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;/&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Y  =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="addPlotButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #111;
border: 1px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
min-width: 80px;</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="clearPlotButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>64</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #111;
border: 1px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
min-width: 80px;</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="resPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>621</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QComboBox" name="lineTypeBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>230</y>
        <width>151</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="savePlotButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>109</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #111;
border: 1px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
min-width: 80px;</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="importPlotButton">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>154</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #111;
border: 1px solid #555;
border-radius: 11px;
padding: 5px;
background: qradialgradient(cx: 0.3, cy: -0.4,
fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #aaa);
min-width: 80px;</string>
      </property>
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
     <widget class="QComboBox" name="plotTypeBox">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>200</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="ColorPicker" name="colorPlotPalette" native="true">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>270</y>
        <width>231</width>
        <height>221</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdison">
    <property name="title">
     <string>Edison</string>
    </property>
   </widget>
   <addaction name="menuEdison"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
